evap/evaluation/models.py

```diff
@@ -388,7 +389,10 @@ def publish(self):
 
     @transition(field=state, source='published', target='reviewed')
     def unpublish(self):
-        pass
+        from evap.results.tools import get_results_cache_key
+        # invalidate cache
+        cache_key = get_results_cache_key(self)
+        cache.delete(cache_key)
```

Comment:
i would prefer the variable inlined, since the method says that it's about to return a cache key. also, the comment adds little value now.